packages/common/src/content/fetch.ts

```diff
@@ -0,0 +1,129 @@
+import { getItem } from "@esri/arcgis-rest-portal";
```

Comment:
this file has the new "always fetch item and enrichments from portal" logic. if you only look at one file in this PR, this should be the one. Also maybe fetch.test.ts